### PR TITLE
Disconnected minor fixes

### PR DESCRIPTION
**Pull request by alycm**
Various small fixes to things I noticed while working on other things.

### CODE REVIEW

File: api/opencl_platform_layer.txt [hunk 1]
```diff
@@ -1385,7 +1386,7 @@ on one or more devices specified in the context.
   | cl_bool
       | Specifies whether the user is responsible for synchronization
         between OpenCL and other APIs.
-        Please refer to the specific sections in the OpenCL 2.0 extension
+        Please refer to the specific sections in the OpenCL 2.2 extension
```
Comment (kpet) [inline api/opencl_platform_layer.txt]: Should we start referring to the unified specification instead?

Comment (alycm) [reply api/opencl_platform_layer.txt]: Yes, I'll reword this.  Those the unified specification is still v2.2-10.

**Comment by bashbaug** [comment]
These changes look good to me.  Did you want to change the text to refer to the unified extension spec?

I don't see a strong reason to wait so I'll accept this PR tomorrow unless you'd rather wait - thanks!

File: api/opencl_platform_layer.txt (hunk 1 above)
Comment (alycm) [reply api/opencl_platform_layer.txt]: Done, I know just say "OpenCL Extension Specification".

**Pull request merged**